jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt

```diff
@@ -3,17 +3,338 @@
+        )
+
+        val codeCatalystBulletsLists: List<BulletAuthPanel> = listOf(
+            BulletAuthPanel(PanelConstants.CANCEL_ICON, message("iam_identity_center.name"), message("aws.getstarted.auth.panel.notSupport_text"), false),
+            BulletAuthPanel(PanelConstants.COMMIT_ICON, message("aws_builder_id.service_name"), "Need to insert tagline"),
+            BulletAuthPanel(PanelConstants.CANCEL_ICON, message("settings.credentials.iam"), message("aws.getstarted.auth.panel.notSupport_text"), false)
+        )
+
+        init {
+
+            val serviceTitleMap: MutableMap<String, List<BulletAuthPanel>> = HashMap()
+            serviceTitleMap.put(message("codewhisperer.experiment"), codeWhispererBulletsLists)
+            serviceTitleMap.put(message("aws.getstarted.resource.panel_title"), resourceBulletsLists)
+            serviceTitleMap.put(message("caws.devtoolPanel.title"), codeCatalystBulletsLists)
+
+            addToCenter(
+                panel {
+                    indent {
+
+                        row {
+                            panelTitle.let { label(it).bold().applyToComponent { foreground = PanelConstants.TITLE_TEXT_FONTCOLOR } }
+                        }
+
+                        val defaultList: List<BulletAuthPanel> = mutableListOf()
+
+                        val bulletsList: List<BulletAuthPanel> = serviceTitleMap[panelTitle] ?: defaultList
+
+                        panelTitle.let {
+                            for (bullet in bulletsList) {
+                                row {
+                                    text(bullet.icon)
+                                    panel {
+                                        row(bullet.titleName) {
+                                        }.rowComment(bullet.comment)
+                                            .enabled(bullet.enable)
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            )
+
+            border = IdeBorderFactory.createRoundedBorder().apply {
+                setColor(PanelConstants.TEXT_FONTCOLOR)
+                preferredSize = Dimension(PanelConstants.PANEL_WIDTH, PanelConstants.BULLET_PANEL_HEIGHT)
+            }
+        }
+    }
+
+    object PanelConstants {
+        const val RESOURCE_EXPLORER_LEARN_MORE = "https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/working-with-aws.html"
+        const val RESOURCE_EXPLORER_SIGNUP_DOC = "https://aws.amazon.com/free/"
+        const val AWS_TOOLKIT_DOC = "https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/welcome.html"
+        const val GITHUB_LINK = "https://github.com/aws/aws-toolkit-jetbrains"
+        const val SHARE_FEEDBACK_LINK = "FeedbackDialog"
+        const val COMMIT_ICON = "<icon src='AllIcons.General.InspectionsOK'/>&nbsp;"
+        const val CANCEL_ICON = "<icon src='AllIcons.CodeWithMe.CwmTerminate'/>&nbsp;"
```

Comment:
i believe this will not work if the codewithme plugin is not enabled

Reply:
I need to investigate more on this, Shall we track this in here [IDE-11561](https://sim.amazon.com/issues/IDE-11561) and tackle this later.